prog2.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
+
+import paramiko
+import threading
+import time
+
+com = raw_input("Please specify the command to run: ")
+
+# Hosts to run the commands on
+hosts=['vm1', 'vm2']
+
+def run_ssh_command(host, command):
+    ssh = paramiko.SSHClient()
+    ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    try:
+        ssh.connect(host)
+        stdin, stdout, stderr = ssh.exec_command(command)
+        print "Command output on %s:" % host
+        stdout=stdout.readlines()
+        for line in stdout:
+            print line
+        ssh.close()
```

Comment:
1. I would suggest to prefix each line of output with hostname from which this output is received.
2. stderr is ignored. Why?
3. No exit code check.
4. Potential out-of-memory issue if executed command generates a lot of MB of data on stdout stream